live/global.ini

```diff
@@ -18643,37 +18643,37 @@ Lorville_Destination_Floor08=Etage 08
-MG_Debug_has_met_false=I have not met this MG (warning: sets false for ALL MGs)
-MG_Debug_has_met_true=I have met this MG (warning: sets true for ALL MGs)
-MG_Debug_last_mission_failed=I failed my last mission
-MG_Debug_last_mission_succeeded=I succeeded at my last mission
-MG_Debug_mission_count_0=Mission giver has no missions
-MG_Debug_mission_count_1=Mission giver has one mission
-MG_Debug_mission_count_2=Mission giver has multiple missions
-MG_Debug_mission_is_lawful_false=The available mission is unlawful
-MG_Debug_mission_is_lawful_true=The available mission is lawful
-MG_Debug_on_mission_false=I am not currently on a mission for MG
-MG_Debug_on_mission_true=I am already on a mission for MG
-MG_Debug_qualify_false=I do not have an invitation to meet this MG
-MG_Debug_qualify_true=I have an invitation to meet this MR
-MG_Debug_start_options=Choose history and mission details
-MG_Debug_start_random=Randomize history and mission details
+MG_Debug_has_done_missions_false=Ich habe nie eine Mission angenommen vom MG
+MG_Debug_has_done_missions_true=Ich habe schon einmal einen Einsatz für dieses MG gewonnen oder verloren
+MG_Debug_has_met_false=Ich habe diesen MG noch nicht getroffen (warning: sets false for ALL MGs)
+MG_Debug_has_met_true=Ich habe bereits diesem MG getroffen (warning: sets true for ALL MGs)
+MG_Debug_last_mission_failed=Ich habe die letze Mission versaut
+MG_Debug_last_mission_succeeded=Ich habe meine letzte Mission Erfolgreich abgeschlossen
+MG_Debug_mission_count_0=Missionsgeber hat keine Missionen
+MG_Debug_mission_count_1=Missionsgeber hat eine Mission
+MG_Debug_mission_count_2=Missionsgeber hat mehrere Missionen
+MG_Debug_mission_is_lawful_false=Die verfügbare Mission ist Rechtswidrig
+MG_Debug_mission_is_lawful_true=Die verfügbare Mission ist Rechtmäßig
+MG_Debug_on_mission_false=Ich bin derzeit nicht im Auftrag von MG tätig.
+MG_Debug_on_mission_true=Ich bin bereits auf einer Mission für MG
+MG_Debug_qualify_false=Ich habe keine Einladung zu einem Treffen mit diesem MG
+MG_Debug_qualify_true=Ich habe eine Einladung zu einem Treffen mit diesem MR
+MG_Debug_start_options=Geschichte und Missionsdetails auswählen
+MG_Debug_start_random=Zufallsgenerierung von Verlauf und Missionsdetails
 MG_Hurston_from=Constantine Hurston
-MG_hurston_invite_desc=PLACEHOLDER
-MG_hurston_invite_from=Hurston - PLACEHOLDER
-MG_hurston_invite_marker=Constantine - PLACEHOLDER
-MG_hurston_invite_obj_long=PLACEHOLDER
-MG_hurston_invite_obj_short=PLACEHOLDER
-MG_hurston_invite_title=Hurston Invite - PLACEHOLDER
-MKTG_CUSTOMS1_CV_Access_Allow_IG_003_YoureClearMove=Du bist klar. Weitermachen.
+MG_hurston_invite_desc=PLATZHALTER
+MG_hurston_invite_from=Hurston - PLATZHALTER
+MG_hurston_invite_marker=Constantine - PLATZHALTER
+MG_hurston_invite_obj_long=PLATZHALTER
+MG_hurston_invite_obj_short=PLATZHALTER
+MG_hurston_invite_title=Hurston Invite - PLATZHALTER
+MKTG_CUSTOMS1_CV_Access_Allow_IG_003_YoureClearMove=Du bist sauber. Weitermachen.
```

Comment:
Diese und andere werden in meinem PR noch mal ersetzt. "Du bist sauber" muss "Du bist frei" lauten. `You're clear. Move along.` :)